src/docs/index.md

```diff
@@ -128,6 +128,96 @@ title: Akka.NET Documentation
 - [Scheduler](Scheduler)
 - [Circuit Breaker](CircuitBreaker)
 
+### Akka Streams (work in progress)
+*we are currently in the progress of porting the streams documentation for Akka.Net. As such not all sections will be available*
+
+- [Introduction](streams/introduction)
+	- [Motivation](streams/introduction#motivation)
+	- [How to read these docs](streams/introduction#how-to-read-these-docs)
+- [Quick Start Guide](streams/quickstart)
+	- [Reusable Piece](streams/quickstart#reusable-pieces)
+	- [Time-Based Processing](streams/quickstart#time-based-processing)
+- [Reactive Tweets](streams/reactivetweets)
+	- [Transforming and consuming simple streams](streams/reactivetweets#transforming-and-consuming-simple-streams)
+	- [Flattening sequences in streams](streams/reactivetweets#flattening-sequences-in-streams)
+	- [Broadcasting a stream](streams/reactivetweets#broadcasting-a-stream)
+	- [Back-pressure in action](streams/reactivetweets#back-pressure-in-action)
+	- [Materialized values](streams/reactivetweets#materialized-values)
+- [Design Principles behind Akka Streams](streams/designprinciples)
+	- [What shall users of Akka Streams expect?](streams/designprinciples#what-shall-users-of-akka-streams-expect)
+	- [Interoperation with other Reactive Streams implementations](streams/designprinciples#interoperation-with-other-reactive-streams-implemenations)
+	- [What shall users of streaming libraries expect?](streams/designprinciples#what-shall-users-of-streaming-libraries-expect)
+	- [The difference between Error and Failure](streams/designprinciples#the-difference-between-Error-and-Failure)
+- [Basics and working with Flows](streams/basics)
+	- [Core concepts](streams/basics#core-concepts)
+	- [Defining and running streams](streams/basics#defining-and-running-streams)
+	- [Back-pressure explained](streams/basics#back-pressure-explained)
+	- [Stream Materialization](streams/basics#stream-materialization)
+	- [Stream ordering](streams/basics#stream-ordering)
+- [Working with Graphs](streams/workingwithgraphs)
+	- [Constructing Graphs](streams/workingwithgraphs#constructing-graphs)
+	- [Constructing and combining Partial Graphs](streams/workingwithgraphs#constructing-and-combining-partial-graphs)
+	- [Constructing Sources, Sinks and Flows from Partial Graphs](streams/workingwithgraphs#constructing-sources-sinks-and-flows-from-partial-graphs)
+	- [Combining sources and Sinks with simplified API](streams/workingwithgraphs#combining-sources-and-sinks-with-simplefied-api)
+	- [Building reusable Graph components](streams/workingwithgraphs#building-reusable-graph-components)
+	- [Predefined shapes](streams/workingwithgraphs#predefined-shapes)
+	- [Bidirectional Flows](streams/workingwithgraphs#bidirectional-flows)
+	- [Accessing the materialized value inside the graph](streams/workingwithgraphs#accessing-the-materialized-value-inside-the-graph)
+	- [Graph cycles, liveness and deadlocks](streams/workingwithgraphs#graph-cycles-liveness-and-deadlocks)
```

Comment:
Warning: I've already found, that one of the examples showing how to solve deadlock problem from this section, doesn't work in our code for some reason (see: https://github.com/akkadotnet/akka.net/pull/1929 ).